.travis.yml

```diff
@@ -1,21 +1,21 @@
 language: php
 
 php:
-  - "5.5"
+  - "7.0"
 
 sudo: required
 
-dist: trusty
-
+dist: xenial
 cache:
   directories:
   - node_modules
 
 before_script:
   - npm install -g grunt-cli
   - npm install
+  - npm install -g casperjs
 
-script: npm run-script ci
+script: npm run-script test
```

Comment:
I believe we don't need to change this.

Reply:
In the ReadMe describes the test process with `npm run-script test`. What is the purpose of `npm run-script ci`?